### PR TITLE
Datamodel overview improvements

### LINKED ISSUE #5586: Data model diagram front-end improvements
# Current Behavior

CleanShot 2024-05-27 at 10 23 07

CleanShot 2024-05-27 at 10 31 36

# Desired Behavior

### Card

1. Increase the border-radius to 8px
2. Add some box-shadow

image

### Card title

1. Increase the spacing between the icon and the name
3. Reduce the font-weight to "Medium"

CleanShot 2024-05-27 at 10 24 00

### Relation Fields

1. Check the spacing
2. Label color should be "Primary"

CleanShot 2024-05-27 at 10 28 04

### Other fields

Color should stay the same. Icon should be a "chevron down"

CleanShot 2024-05-27 at 10 30 28

### Buttons

We should use the `iconButtonGroup` and Tabler icons

CleanShot 2024-05-27 at 10 33 47

Shadow on hover: 

CleanShot 2024-05-27 at 10 34 04

### PR DESCRIPTION
**Pull request by brendanlaschke**
closes #5586
Bildschirmfoto 2024-06-07 um 14 05 39

### CODE REVIEW

File: packages/twenty-front/src/modules/settings/data-model/graph-overview/components/SettingsDataModelOverview.tsx [hunk 1]
```diff
@@ -236,10 +239,34 @@ export const SettingsDataModelOverview = () => {
...
         nodeTypes={NodeTypes}
         onNodesChange={handleNodesChange}
+        nodesDraggable={isInteractive}
+        elementsSelectable={isInteractive}
         proOptions={{ hideAttribution: true }}
       >
         <Background />
-        <Controls />
+
+        <IconButtonGroup
```
Comment (FelixMalfait) [inline packages/twenty-front/src/modules/settings/data-model/graph-overview/components/SettingsDataModelOverview.tsx:248]: Should we had a shadow on that? I don't think I see it on my side

**Review by FelixMalfait — APPROVED** [review]
This looks great thank you!!! One minor comment

File: packages/twenty-front/src/modules/settings/data-model/graph-overview/components/SettingsDataModelOverview.tsx (hunk 1 above)
Comment (brendanlaschke) [reply packages/twenty-front/src/modules/settings/data-model/graph-overview/components/SettingsDataModelOverview.tsx:245]: remainder in the general react flow css above :) should be fixed now

**Review by FelixMalfait — APPROVED** [review]

File: packages/twenty-front/src/modules/settings/data-model/graph-overview/components/SettingsDataModelOverview.tsx (hunk 1 above)
Comment (FelixMalfait) [reply packages/twenty-front/src/modules/settings/data-model/graph-overview/components/SettingsDataModelOverview.tsx:207]: Thanks!

**Pull request merged**